pylintrc

```diff
@@ -287,7 +286,6 @@ disable =
 	illegal-waffle-usage,
 	
 	logging-fstring-interpolation,
-	django-not-available,
```

Comment:
This was the hand-edited line that is now missing from pylintrc, and is causing the linting failures.

Reply:
ah thanks, I'll add that to `_tweaks`